app/controllers/paginable/plans_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Paginable::PlansController < ApplicationController
+  include Paginable
+  #before_action :verify_authorized
+  # /paginable/plans/privately_visible/:page
+  def privately_visible
+    # TODO authorize
+    if params[:page] == 'ALL'
+      plans = current_user.active_plans
+    else
+      plans = current_user.active_plans.page(params[:page])
+    end
+    paginable_renderise(partial: 'privately_visible', scope: plans)
+  end
+  # GET /paginable/plans/organisationally_or_publicly_visible/:page
+  def organisationally_or_publicly_visible
+    # TODO authorize
+    if params[:page] == 'ALL'
+      plans = Plan.organisationally_or_publicly_visible(current_user)
+    else
+      plans = Plan.organisationally_or_publicly_visible(current_user).page(params[:page])
+    end
+    paginable_renderise(partial: 'organisationally_or_publicly_visible', scope: plans)
```

Comment:
We need to create controller actions for any model we want to paginate. That's the reason I created the namespace paginable/ since any link used for paginable stuff (e.g. View All, View Less, First, 1...N, Last) when clicked, sends an AJAX request to an specific action under the scope of paginable controllers. 

I'm going to try to be as DRY as possible (e.g. not checking ALL in both places paginable_renderise and in each controller action...